packages/ui-app/src/Input.tsx

```diff
@@ -90,11 +96,32 @@ export default class Input extends React.PureComponent<Props, State> {
           {icon}
           {children}
         </SUIInput>
+        <Notification
+          error={error}
+          info={info}
+          warn={warn}
+        />
       </Labelled>
     );
   }
 
   onChange = (event: React.SyntheticEvent<Element>, { value }: SUIEvent): void => {
     this.props.onChange(value);
   }
+
+  onKeyDown = (event: SUIEvent): void => {
+    const { onKeyDown } = this.props;
+
+    if (onKeyDown) {
+      onKeyDown(event);
```

Comment:
Here grab the values from event (i.e. shift state, key pressed) and pass only those up.

Reply:
I've had to pass up the `event` itself in Input.tsx:
```
    if (onKeyDown) {
      onKeyDown(event);
...
```

I can't pass up destructured properties of event with the following in Input.tsx, because I'd also have to destructure and pass up the `preventDefault` method of `event`, which I haven't been able to do
```
    if (onKeyDown) {
      onKeyDown(key, preventDefault, shiftKey, (target as HTMLInputElement).value);
...
```